spring-boot-admin-server-ui/src/main/frontend/views/instances/httptrace/index.vue

```diff
@@ -265,7 +266,13 @@ const addToFilter = (oldFilter, addedFilter) =>
       createSubscription() {
         const vm = this;
         return timer(0, 5000)
-          .pipe(concatMap(vm.fetchHttptrace))
+          .pipe(concatMap(vm.fetchHttptrace), retryWhen(
+            err => {
+              return err.pipe(
+                delay(1000),
+                take(2)
```

Comment:
in the previous implementation you chose `take(5)` instead of `2`. Is there any reason for doing this? Would it make sense to refactor the whole `pipe` implementation to a function which can be reused anywhere? Like as follows:

```
const pipeRetry = (stream) => pipe(concatMap(stream), retryWhen(
    err => {
      return err.pipe(
        delay(1000),
        take(5)
      )
    }));

return timer(0,5000).pipeRetry(vm.fetchHttptrace).subscribe(....);
```